pkg/application/service/services.go

```diff
@@ -36,7 +37,7 @@ type VerificationService interface {
 }
 
 type MemberClusterService interface {
-	GetClusterAccess(userID, username, workspace string) (*access.ClusterAccess, error)
+	GetClusterAccess(userID, username, workspace, proxyName string) (*access.ClusterAccess, error)
```

Comment:
Can we call this `pluginName` or `proxyPluginName` instead? `proxyName` could be confusing. There is only one proxy. Just multiple plugins.